packages/api-explorer/src/scenes/DiffScene/diffUtils.ts

```diff
@@ -118,3 +118,19 @@ export const diffToSpec = (
   result.types = {}
   return result
 }
+
+/**
+ * Returns all valid diff options contained in list
+ * @param opts url diff options parameter value
+ */
+export const getValidDiffOptions = (opts: string | null) => {
```

Comment:
can opts just be made an optional string arg here? is `null` explicitly needed?

Reply:
I pass in `searchParams.get('opts')` in both DiffScene and diffStoreSync. When the param doesn't exist in the URL it will return null, so this was just a way to be able to handle it without doing any explicit checks before passing it in as input.